app/lib/frontend/dom/dom.dart

```diff
@@ -442,13 +442,22 @@ class Image {
   final String alt;
   final int? width;
   final int? height;
+  final String? role;
 
   Image({
     required this.src,
     required this.alt,
     required this.width,
     required this.height,
+    this.role,
   });
+
+  Image.presentation({
```

Comment:
I like this!

Could you add a dartdoc on what the behavior is?

Reply:
Updated. Also renamed to `Image.decorative`, as the image itself is decorative, its role is presentation.